documentation/book/cluster-operator.adoc

```diff
@@ -132,11 +136,11 @@ data:
       "lowercaseOutputName": true,
       "rules": [
           {
-            "pattern": "kafka.server<type=(.+), name=(.+)PerSec\\w*><>Count",
+            "pattern": "kafka.server&lt;type=(.+), name=(.+)PerSec\\w*&gt;&lt;&gt;Count",
```

Comment:
Is this really necessary? Doesn't the ascidoc HTML rendering do this quoting for you?

Reply:
Well, asciidoc is able to handle this. But `ccutil` used downstream isn't.